docs/gitbook/install/flagger-install-on-alibaba-servicemesh.md

```diff
@@ -4,133 +4,54 @@ This guide walks you through setting up Flagger on Alibaba ServiceMesh.
-- job_name: 'istio-mesh'
-  kubernetes_sd_configs:
-  - role: endpoints
-    namespaces:
-      names:
-      - istio-system
-  relabel_configs:
-  - source_labels: [__meta_kubernetes_service_name, __meta_kubernetes_endpoint_port_name]
-    action: keep
-    regex: istio-telemetry;prometheus
-# Scrape config for envoy stats
-- job_name: 'envoy-stats'
-  metrics_path: /stats/prometheus
-  kubernetes_sd_configs:
-  - role: pod
-  relabel_configs:
-  - source_labels: [__meta_kubernetes_pod_container_port_name]
-    action: keep
-    regex: '.*-envoy-prom'
-  - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
-    action: replace
-    regex: ([^:]+)(?::\d+)?;(\d+)
-    replacement: $1:15090
-    target_label: __address__
-  - action: labeldrop
-    regex: __meta_kubernetes_pod_label_(.+)
-  - source_labels: [__meta_kubernetes_namespace]
-    action: replace
-    target_label: namespace
-  - source_labels: [__meta_kubernetes_pod_name]
-    action: replace
-    target_label: pod_name
-- job_name: 'istio-policy'
-  kubernetes_sd_configs:
-  - role: endpoints
-    namespaces:
-      names:
-      - istio-system
-  relabel_configs:
-  - source_labels: [__meta_kubernetes_service_name, __meta_kubernetes_endpoint_port_name]
-    action: keep
-    regex: istio-policy;http-policy-monitoring
-- job_name: 'istio-telemetry'
-  kubernetes_sd_configs:
-  - role: endpoints
-    namespaces:
-      names:
-      - istio-system
-  relabel_configs:
-  - source_labels: [__meta_kubernetes_service_name, __meta_kubernetes_endpoint_port_name]
-    action: keep
-    regex: istio-telemetry;http-monitoring
-- job_name: 'pilot'
-  kubernetes_sd_configs:
-  - role: endpoints
-    namespaces:
-      names:
-      - istio-system
-  relabel_configs:
-  - source_labels: [__meta_kubernetes_service_name, __meta_kubernetes_endpoint_port_name]
-    action: keep
-    regex: istiod;http-monitoring
-  - source_labels: [__meta_kubernetes_service_label_app]
-    target_label: app
-- job_name: 'sidecar-injector'
-  kubernetes_sd_configs:
-  - role: endpoints
-    namespaces:
-      names:
-      - istio-system
-  relabel_configs:
-  - source_labels: [__meta_kubernetes_service_name, __meta_kubernetes_endpoint_port_name]
-    action: keep
-    regex: istio-sidecar-injector;http-monitoring
-```
+In the Alibaba Cloud Service Mesh (ASM) console, click  Settings to enable the collection of Prometheus monitoring metrics. You can use the self-built Prometheus monitoring, or you can use the Alibaba Cloud ARMS Prometheus monitoring plug-in that has joined the ACK cluster, and use ARMS Prometheus to collect monitoring indicators.
 
 ## Install Flagger
 
 Add Flagger Helm repository:
 
 ```bash
 helm repo add flagger https://flagger.app
-helm repo update
 ```
 
 Install Flagger's Canary CRD:
 
-```yaml
-kubectl apply -f $FLAGGER_SRC/artifacts/flagger/crd.yaml
+```bash
+kubectl apply -f https://raw.githubusercontent.com/fluxcd/flagger/v1.21.0/artifacts/flagger/crd.yaml
```

Comment:
The latest release of Flagger is now `v1.22.0`. Maybe it'd be good to have some kind of

```cli
VERSION=$(curl -s "https://api.github.com/repos/fluxcd/flagger/releases" | jq -r '.[] | .tag_name' | sort -V | tail -n 1)
```

logic here?